packages/driver/src/cy/net-stubbing/events/before-request.ts

```diff
@@ -302,7 +302,10 @@ export const onBeforeRequest: HandlerFn<CyHttpMessages.IncomingRequest> = (Cypre
     resolve = _resolve
   })
 
-  request.setLogFlag = Cypress.ProxyLogging.logInterception(request, route).setFlag
+  request.setLogFlag = Cypress.RequestLogs['getFlagSetter'](request, route)
```

Comment:
```suggestion
  request.setLogFlag = Cypress.RequestLogs.getFlagSetter(request, route)
```

Reason to use string-style property access?

Reply:
I can't quite figure out how to get the types to work here. After adding `Cypress.RequestLogs` to `cypress.d.ts`, it seems to override the `Cypress.RequestLogs` declaration in the driver's internal type instead of merging with it. Since `getFlagSetter` isn't on the public API, it's no longer typed... this is my hack to get around that. Any idea how best to fix this?

Reply:
Rip TypeScript out of the repository, cutting our codebase by 15% and our compilation time by 25%? :stuck_out_tongue: 

But more seriously, `// @ts-ignore` would be my go-to. Or leave it as is. :woman_shrugging:

Reply:
:scream: 

I changed it to `// @ts-ignore` and added a clarifying comment/plea for help in a79b9435f01bcfa69f20348b001a449164ae91ae